boards/opentitan/README.md

```diff
@@ -70,56 +70,113 @@ Opentitan is supported on both an FPGA and in Verilator. Slightly different
 versions of the EarlGrey chip implementation are required for the different
 platforms. By default the kernel is compiled for the FPGA.
 
-## Setting up Verilator
-
-For a full guide see the official OpenTitan documentation: https://docs.opentitan.org/doc/ug/getting_started_verilator/
-
-A quick summary on how to do this is included below though
-
-### Build FuseSoc
+### Setup OpenTitan
 
 ```shell
 git clone https://github.com/lowRISC/opentitan.git
 cd opentitan
+
+# Use the OpenTitan_SHA currently supported by Tock
 git checkout <OpenTitan_SHA>
 pip3 install --user -r python-requirements.txt
+```
+Make sure to follow [OpenTitan getting started instructions](https://docs.opentitan.org/doc/getting_started/) to setup required dependencies/toolchains.
+
+### **Fedora dependencies quick install**
+
+Note: the OpenTitan documentation provides an easy installation for packages for Ubuntu based distributions. This is an equivalent command to install the (mostly) same packages for Fedora users.
 
-LANG="en_US.UTF-8" fusesoc --cores-root . run --flag=fileset_top --target=sim --setup --build lowrisc:dv:chip_verilator_sim
+```shell
+sudo dnf install autoconf bison make automake gcc gcc-c++ kernel-devel \
+		 clang-tools-extra clang cmake curl \
+		 doxygen flex g++ git golang lcov elfutils-libelf \
+ 		 libftdi libftdi-devel ncurses-compat-libs openssl-devel \
+		 systemd-devel libusb redhat-lsb-core \
+		 make ninja-build perl pkgconf python3 python3-pip python3-setuptools \
+		 python3-urllib3 python3-wheel srecord tree xsltproc zlib-devel xz clang-tools-extra \
+		 clang11-libs clang-devel elfutils-libelf-devel
 ```
 
-### Build Boot Rom/OTP Image
+## Setting up Verilator
+
+For a full guide see the official [OpenTitan Verilator documentation](https://docs.opentitan.org/doc/ug/getting_started_verilator/)
+
+A quick summary on how to do this is included below though
+
+### Build Boot Rom/OTP Image and FuseSOC
+
+Build **only the targets** we care about. Note: the following commands assume  `bazelisk.sh` has been aliased to `bazel`. You may do so with `alias bazel="PATH_TO/bazelisk.sh"` (from the root of the OpenTitan directory).
 
-Build only the targets we care about.
 ```shell
-./meson_init.sh
-ninja -C build-out sw/device/lib/testing/test_rom/test_rom_export_sim_verilator
-ninja -C build-out sw/device/otp_img/otp_img_sim_verilator.vmem
+# To build the ROM
```

Comment:
Please call this the "test ROM" -- "ROM" alone refers to the production ROM.

Reply:
Fixed!